common/test/acceptance/pages/lms/programs.py

```diff
@@ -31,3 +31,32 @@ class ProgramDetailsPage(PageObject):
 
     def is_browser_on_page(self):
         return self.q(css='.js-program-details-wrapper').present
+
+
+class ProgramMarketingPage(PageObject):
+    """Program marketing page."""
+    program_uuid = str(uuid4())
+    url = '{base}/programs/{program_uuid}/about'.format(base=BASE_URL, program_uuid=program_uuid)
+
+    def is_browser_on_page(self):
+        return self.q(css='#program-details-page').present
+
+    @property
+    def are_course_cards_present(self):
```

Comment:
These properties are only used in one place. Do they really need to be properties, or can they just be calls in the consuming test?